scripts/create_perf_json.py

```diff
@@ -1333,7 +1381,9 @@ def to_perf_json(self, outdir: str):
             _verboseprint2(f'Generating {event_type} events from {self.files[event_type]}')
             with urllib.request.urlopen(self.files[event_type]) as event_json:
                 json_data = json.load(event_json)
-                pmon_events = [PerfmonJsonEvent(x) for x in json_data['Events']]
+                # UNC_IIO_BANDWIDTH_OUT events are broken on Linux so skip if they exist.
```

Comment:
Would a TODO be a helpful reminder if this is something that Linux might fix in the future?

Reply:
Kan knows a fuller story, I believe they were available, found to be broken, removed and that was all some time ago - the events exist only for icelakex and snowridgex. I need to drop the events here as otherwise a perf test will fail when trying to use them and they are found not to exist.

Reply:
OK, Thanks for the details.

Reply:
I've updated this test for SPR where the bandwidth out events are present